cookbooks/allken_et_al_2012_rift_interaction/allken.prm

```diff
@@ -0,0 +1,147 @@
+# input file to replicate Allken et al, G3, 2012.
```

Comment:
```suggestion
# Input file to replicate Allken et al, G3, 2012.
```

Reply:
done

Reply:
sorry @anne-glerum  and @bangerth , I thought I had addressed all comments. I have added the plastic damper params, but I honestly don't feel like testing their influence right now, it is outside the scope of a cookbook I think